app/src/main/java/com/sunasterisk/youtubebackground/utils/VideoUtils.kt

```diff
@@ -0,0 +1,22 @@
+package com.sunasterisk.youtubebackground.utils
+
+object VideoUtils {
+    fun formatLikeCount(likeCount: Long?): String{
+        if (likeCount == null) {
+            return "00000"
+        }
+        var suffixChar = ""
+        var prefixChar = ""
+
+        if (likeCount!! / 1000000000 > 0) {
```

Comment:
nếu check null rồi thì đoạn này nó sẽ tự hiểu 
```suggestion
        if (likeCount / 1000000000 > 0) {
```